src/core/services/theming/theming.js

```diff
@@ -510,6 +510,7 @@ function ThemingProvider($mdColorPalette, $$mdMetaProvider) {
 
       self[colorType + 'Color'] = function() {
         var args = Array.prototype.slice.call(arguments);
+        // eslint-disable-next-line no-console
```

Comment:
This doesn't look like a test. We always want no-console because Angular provides a log service